whatlies/embeddingset.py

```diff
@@ -746,23 +755,23 @@ def plot_3d(
         else:
             x_axis_metric = Embedding._get_plot_axis_metric_callable(x_axis_metric)
             x_val = self.compare_against(x_axis, mapping=x_axis_metric)
-            x_lab = x_axis.name
+            x_lab = x_label if x_label else x_axis.name
```

Comment:
This line should be put after the `if/else`, otherwise it won't be effective when `if` condition is True. Further it should be like `x_lab = x_label if x_label is not None else ....`, otherwise when `x_label = ''` this would not be effective (someone may want to not use a label for an axis using this trick). The same goes for labels of other axes.